example/config/healthcheck/apichecks.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
         {
             type: 'json',
             name: 'CNN Homepage',
-            url: 'http://www.cnn.com/_healthcheck',
+            url: 'https://bulletin.cnn.com/_health',
```

Comment:
Will this effect the harbor healthcheck?

Reply:
it shouldn't... I'll use mailcar instead